src/auto-evo/selection_pressure/PredationEffectivenessPressure.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace AutoEvo;
+
+public class PredationEffectivenessPressure : SelectionPressure
+{
+    public readonly Species Prey;
```

Comment:
Does this have to be public?

Reply:
Yes, it is checked aginst in ModifyExistingSpeices as a way to get the preadators of a creature.